app/models/wpcc/salary_message.rb

```diff
@@ -0,0 +1,23 @@
+module Wpcc
+  class SalaryMessage
+    include ActiveModel::Model
+    attr_accessor :salary, :salary_frequency, :message
+
+    TAX_RELIEF_THRESHOLD_RATE = {
+      year: 11_500,
+      month: 958.33,
+      fourweeks: 884.61,
+      week: 221.15
+    }.freeze
+
+    def tax_relief_warning?
+      message == :tax_relief_warning && salary_below_tax_relief_threshold?
+    end
```

Comment:
This is the method that verified in the view to display to the user the message warning. I would move this method to the Presenter and make the salary below tax public here (because it is the public interface of displaying one type of a salary message. Wdyt?